### DISCUSSION TITLE
Only searches nvidia GPU's.

### DISCUSSION THREAD

**Discussion by Peanutbutterwaffles** [jef/streetmerchant#1084] [Help]
There's nothing in my .env for nvidia anymore.
I only have 
"SHOW_ONLY_BRANDS="asus,gigabyte,amd reference"
SHOW_ONLY_MODELS="rog strix,tuf,xfx,msi,gigabyte"
SHOW_ONLY_SERIES="rx6800xt,rx6900xt" 

Not sure why my selected store is :nvidia 
How do i change my selected store? Why will it only show me Nvidia GPU's?

**Comment by thefl0yd** [comment]
What's the leading quote on your first line:
"SHOW_ONLY_BRANDS="...
^
|----- is this a copy/paste error? If not, there's your problem. Get rid of it.

**Comment by Peanutbutterwaffles** [comment]
Yeah thats a copy paste error.
